HousingSearchApi/data/elasticsearch/transactionsIndex.json

```diff
@@ -0,0 +1,188 @@
+{
+  "aliases": {},
+  "mappings": {
+    "properties": {
+      "targetId": {
+        "type": "text"
+      },
+      "targetType": {
+        "type": "keyword",
+        "normalizer": "my_normalizer",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "periodNo": {
+        "type": "short"
+      },
+      "financialYear": {
+        "type": "short"
+      },
+      "financialMonth": {
+        "type": "short"
+      },
+      "transactionSource": {
+        "type": "text"
+      },
+      "transactionType": {
+        "type": "text"
+      },
+      "transactionDate": {
+        "type": "keyword",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "transactionAmount": {
+        "type": "keyword",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "paymentReference": {
+        "type": "keyword",
+        "normalizer": "my_normalizer",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "bankAccountNumber": {
```

Comment:
Is bank account number something that is needed for searching/displaying search results?